.cspell.json

```diff
@@ -16,6 +16,7 @@
     "descheduler",
     "encodedkey",
     "envs",
+    "gcpckms",
```

Comment:
I don't recall our conversation very well, but I wonder why cloud-vendor stuff is here?

Reply:
you need to add this to ignored words. Otherwise spell raise an error

Reply:
If I read it back my point didn't make a lot of sense, sorry about that. I understand the logic from .cspell of course. But I thought (in general) we would base our secret management not on any KMS provider but only on a cluster-by-cluster basis. 

I don't have to go full review mode on this since @Morriz is already doing that I see. I was just curious about this.